### PR TITLE
feat: bump chakra packages to latest

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

cc @reallybeard this includes two releases that may be relevant to the issue with the DeFi's `<NestedListItem />`'s :before pseudo-element we're having:

- https://chakra-ui.com/changelog/v2.5.3
> Add support for scoping the css reset to specific selector.
- https://chakra-ui.com/changelog/v2.7.0
> Add support for disabling global styles via the disableGlobalStyle prop on the ChakraProvider.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Style might be borked across the app

## Testing

- Do a smallish smoke test navigating around the app and ensure nothing looks visually odd

### Engineering

- ☝🏽 

### Operations

- ☝🏽

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5030** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5030" target="_blank">Graphite</a> 
    * **PR #5031** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5031" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5031?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -74,12 +74,12 @@
...
-    "@chakra-ui/system": "^2.5.8",
-    "@chakra-ui/tag": "^2.0.2",
-    "@emotion/react": "^11.9.3",
-    "@emotion/styled": "^11.9.3",
+    "@chakra-ui/icons": "^2.1.0",
+    "@chakra-ui/react": "^2.8.0",
+    "@chakra-ui/system": "^2.6.0",
+    "@chakra-ui/tag": "^3.1.0",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
```
Comment (gomesalexandre) [inline package.json:82]: Not purely chakra related, but ensures we get the latest emotion minor.
This is only really used by the wherever plugin:

image

File: package.json [hunk 2]
```diff
@@ -187,7 +187,7 @@
     "simplex-noise": "^3.0.1",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "styled-components": "^5.3.6",
+    "styled-components": "^6.0.7",
```
Comment (gomesalexandre) [inline package.json:190]: Not purely chakra related, but ensures we get the latest emotion minor.
This is only really used by the wherever plugin

```shell
❯ npm why styled-components
styled-components@6.0.7
node_modules/styled-components
  peer styled-components@"*" from @styled-icons/octicons@10.47.0
  node_modules/@styled-icons/octicons
    @styled-icons/octicons@"^10.47.0" from @wherever/react-notification-feed@0.4.4
    node_modules/@wherever/react-notification-feed
      @wherever/react-notification-feed@"^0.4.4" from the root project
  peer styled-components@">=5" from @wherever/react-notification-feed@0.4.4
  node_modules/@wherever/react-notification-feed
    @wherever/react-notification-feed@"^0.4.4" from the root project
  styled-components@"^5.3.6" from the root project
```

image

File: package.json (hunk 1 above)
Comment (gomesalexandre) [inline package.json:82]: Actually chakra related:

```
❯ npm why @emotion/styled @emotion/react | grep node.modules..chakra.ui | sed 's/^ *//' | sort -u
node_modules/@chakra-ui/accordion
node_modules/@chakra-ui/alert
node_modules/@chakra-ui/avatar
node_modules/@chakra-ui/breadcrumb
node_modules/@chakra-ui/button
node_modules/@chakra-ui/card
node_modules/@chakra-ui/checkbox
node_modules/@chakra-ui/close-button
node_modules/@chakra-ui/control-box
node_modules/@chakra-ui/css-reset
node_modules/@chakra-ui/editable
node_modules/@chakra-ui/form-control
node_modules/@chakra-ui/icon
node_modules/@chakra-ui/icons
node_modules/@chakra-ui/image
node_modules/@chakra-ui/input
node_modules/@chakra-ui/layout
node_modules/@chakra-ui/media-query
node_modules/@chakra-ui/menu
node_modules/@chakra-ui/modal
node_modules/@chakra-ui/number-input
node_modules/@chakra-ui/pin-input
node_modules/@chakra-ui/popover
node_modules/@chakra-ui/progress
node_modules/@chakra-ui/provider
node_modules/@chakra-ui/radio
node_modules/@chakra-ui/react
node_modules/@chakra-ui/select
node_modules/@chakra-ui/skeleton
node_modules/@chakra-ui/skip-nav
node_modules/@chakra-ui/slider
node_modules/@chakra-ui/spinner
node_modules/@chakra-ui/stat
node_modules/@chakra-ui/stepper
node_modules/@chakra-ui/switch
node_modules/@chakra-ui/system
node_modules/@chakra-ui/table
node_modules/@chakra-ui/tabs
node_modules/@chakra-ui/tag
node_modules/@chakra-ui/textarea
node_modules/@chakra-ui/toast
node_modules/@chakra-ui/tooltip
node_modules/@chakra-ui/visually-hidden
```

**Review by 0xApotheosis — APPROVED** [review]
Smoke test against production seemed fine, noticed no regressions with this bump.

**Pull request merged**